basics/json-validation/README.md

```diff
@@ -15,5 +15,13 @@ Actix-web features illustrated here include:
     3. chaining futures into a single response used by an asynch endpoint
 
 
+### server
+
+```bash
+cd examples/basics/json-validation
```

Comment:
I guess `cd` should start from the root of the repository, so `cd basics/json-validation`.